.gitmodules

```diff
@@ -1,6 +1,7 @@
 [submodule "qemu"]
 	path = qemu
-	url = https://github.com/pranith/qemu
+	branch = qsim_v25_qemu_26 
+	url = https://github.com/B-Bimmermann/qemu-1.git
```

Comment:
Can you remove this change? This will not be necessary once we merge your qsim_v26 branch into mine.